modules/lambdas/variables.tf

```diff
@@ -54,15 +54,6 @@ variable "db_connect_info_secret_arn" {
   description = "Secret ARN of the AWS secretsmanager secret for connecting to the database."
 }
 
-variable "orca_default_recovery_type" {
-  type        = string
-  description = "The Tier for the restore request. Valid values are 'Standard'|'Bulk'|'Expedited'."
-  validation {
-    condition     = contains(["Standard", "Bulk", "Expedited"], var.orca_default_recovery_type)
-    error_message = "Valid values are 'Standard'|'Bulk'|'Expedited'."
```

Comment:
I think it might be easier to only validate at the top-level to avoid having to update this condition in multiple locations. If people think otherwise, I'll add this validation check in and propagate the storage_class validation down as well.

Reply:
I concur that all those items should be at the top level and not throughout.